### PR TITLE
change instance type for pgbouncer machines

### PR DESCRIPTION
**Pull request by shyamkumarlchauhan**
https://dimagi-dev.atlassian.net/browse/SAAS-14437

##### Environments Affected
Production

### CODE REVIEW

File: environments/production/terraform.yml [hunk 1]
```diff
@@ -310,63 +310,63 @@ servers:
...
+    server_instance_type: t3.large
     network_tier: "db-private"
     az: "c"
     volume_size: 80
     volume_encrypted: yes
     group: "postgresql"
     os: bionic 
-  - server_name: "pgbouncer10-production"
+  - server_name: "pgbouncer10-production" # pgbouncer_host for ucr dbs
     server_instance_type: m5n.large
```
Comment (dannyroberts) [inline environments/production/terraform.yml:362]: Did you mean to include this one too?

File: environments/production/terraform.yml [hunk 2]
```diff
@@ -310,63 +310,63 @@ servers:
     group: "kafka"
     os: bionic
 
-  - server_name: "pgbouncer_a1-production" # host and pgbouncer_host for pl/proxy, pgbouncer host for shard dbs
+  - server_name: "pgbouncer_a1-production" # host and pgbouncer_host for pl/proxys
```
Comment (dannyroberts) [inline environments/production/terraform.yml:313]: This is great, thanks for fixing these comments!

File: environments/production/terraform.yml (hunk 1 above)
Comment (shyamkumarlchauhan) [reply environments/production/terraform.yml:362]: This is being handled in another PR as it is a candidate to be rebuilt in another AZ

Comment (shyamkumarlchauhan) [reply environments/production/terraform.yml:362]: https://github.com/dimagi/commcare-cloud/pull/5807

**Review by dannyroberts — APPROVED** [review]

**Pull request merged**